File: kyu 8/python/filterOutTheGeese.js
function gooseFilter(birds) {
  var geese = ["African", "Roman Tufted", "Toulouse", "Pilgrim", "Steinbacher"];
  answer = [];
  for (let i = 0; i < birds.length; i++) {
    if (geese.indexOf(birds[i]) == -1) answer.push(birds[i]);
  }
  return answer;
}

// bird for bird in birds if bird not in geese

console.log(
  gooseFilter([
    "Mallard",
    "Hook Bill",
    "African",
    "Crested",
    "Pilgrim",
    "Toulouse",
    "Blue Swedish",
  ])
);

File: kyu 8/python/filter_out_the_geese.py
def exist(var):
    geese = ["African", "Roman Tufted", "Toulouse", "Pilgrim", "Steinbacher"]
    if var in geese:
        return False
    else:
        return True


def goose_filter(birds):
    filtered = filter(exist, birds)
    return(list(filtered))


print(goose_filter(["Mallard", "Hook Bill", "African",
                    "Crested", "Pilgrim", "Toulouse", "Blue Swedish"]))


# # master
# geese = {"African", "Roman Tufted", "Toulouse", "Pilgrim", "Steinbacher"}

# def goose_filter(birds):
#     return [bird for bird in birds if bird not in geese]
